### ISSUE TITLE
A11y_mssql for VSCode_Message Window_ScreenReader:Screen reader is announcing the "message" dropdown as collapsed/expanded state only for the first time.

### ISSUE THREAD

**Issue by Roopesh-T** [microsoft/vscode-mssql#17199]
## Actual:
Screen reader is announcing the "message" dropdown as collapsed/expanded state only for the first time.

**Similar issue is observed with Jaws**
**Similar issue is also observed in Select top 1000 Results window_Results and messages window.**

## Expected:
Screen reader should announce the "message" dropdown as collapsed/expanded state for as many times as the user uses the control.

**GitHubTags:**#A11yMAS;#A11yTCS;#BM_mssql for VSCode_Jan2022;#mssql for VSCode;#Win32;#FTP;#DesktopApp;#A11ySev2;#AINotInScope;#WCAG1.3.1;#NVDA;#Jaws;#STP;#Element:Dropdown;

## Environment Details:
Application Name: mssql for VSCode
Windows Version: Win11
Screen Reader: NVDA(2021.3), Jaws(2021.2103.174)

## Repro Steps:

1. Open Visual studio code 
2. Tab till extension button and search for "mssql"
3. Tab till install and move to the SQL Server  button
4. Tab till Add Connection Button and insert the valid credentials
5. Right click on the server/database node you want to run the new query on.
6. Press enter “New Query” write any valid query and click on "Execute query button"
7. Message Window will open
8. Run NVDA and tab to the "message" dropdown and check if the NVDA is announcing "message" dropdown as collapsed/expanded state only for the first time.

## User Impact:
Screen reader users will not get the collapsed/expanded state info from the second time.

“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”
## Attachments
![17199](https://user-images.githubusercontent.com/86601127/149271036-d2dad955-f6f1-4c65-9dbc-86a213e1f3a9.gif)
![17199 2](https://user-images.githubusercontent.com/86601127/149270053-bf9a3c1e-358b-490b-a6a6-6d1bb26b3fa1.gif)

**Comment by smartguest** [comment]
This is the case with the rest of the vscode dropdowns, the name of the tab followed by the current state, and when unfocused and focused again it will announce it again. 

However, unlike the file explorer dropdown in vscode, when the state of the tab is changed, it's not announced at all which is very concerning as a user might not know if the tab was just expanded/collapsed when clicked or not. There needs to be some of the same code from the explorer dropdown for handling the state added to this dropdown if its specific to vscode-mssql or perhaps vscode itself if the extension uses a particular dropdown class from it.

**Comment by smartguest** [comment]
If you click on the header and you scroll the mouse down a bit, the screen reader will correctly announce whether the results/message pane is expanded or not, this is going to be tricky to fix as it requires positioning knowledge

**Comment by smartguest** [comment]
Closing issue as its more of an individual screen reader issue:

Microsoft narrator does not announce whether the item is expanded or collapsed when the header is interacted with. 

We have followed the proper specification by adding the aria-expanded and collapsed tags to the header, and the only change that made NVDA read the status when the element is expanded or collapsed is when we removed the css background-image changing (for the dropdown icon). this still won't work with Microsoft Narrator unfortunately.

Issue to file with NVDA: 

when there's two separate CSS background image styles with the properties: 

background-image: (url 1 for expanded and url 2 for collapsed)
background-repeat:no-repeat;
background-position: 2px;

when an item is collapsed or expanded and the CSS background image changes based on its class, the screen reader will announce the item only once.

**Comment by Roopesh-T** [comment]
As mentioned in the above comment on Feb 5th "Issue to file with NVDA and JAWS if applicable", Please share the Sample app where the issue repro.
This bug can be closed by trusted testers by C + AI team after bug verification. Can you please change the bug to open state.

**Comment by Roopesh-T** [comment]
![17199_Closed](https://user-images.githubusercontent.com/86601127/216285714-a1bb2285-f12e-477c-a64f-6874db260ba1.gif)
Issue does not repro so closing the bug.
